pkg/adapter/awsperformanceinsightssource/adapter.go

```diff
@@ -165,9 +175,8 @@ func (a *adapter) PollMetrics(priorTime time.Time, currentTime time.Time) {
 				}
 
 				event := cloudevents.NewEvent(cloudevents.VersionV1)
-				event.SetType(v1alpha1.AWSEventType(a.arn.Service, v1alpha1.AWSPerformanceInsightsGenericEventType))
+				event.SetType(v1alpha1.AWSPerformanceInsightsGenericEventType)
 				event.SetSource(*d.Key.Metric)
```

Comment:
Maybe the source could also be the ARN now, like in other AWS sources.
And you can use an extension for the metric (`ceer.SetExtension("pimetric", ...)`) in case people want to filter on that.